test/e2e/fixtures.go

```diff
@@ -107,33 +117,33 @@ func skipIfNoWindowsNodes(tb testing.TB) {
 	}
 }
 
-func skipIfFeatureDisabled(tb testing.TB, data *TestData, feature featuregate.Feature, checkAgent bool, checkController bool) {
+func skipIfFeatureDisabled(tb testing.TB, feature featuregate.Feature, checkAgent bool, checkController bool) {
 	if checkAgent {
-		if featureGate, err := data.GetAgentFeatures(antreaNamespace); err != nil {
+		if featureGate, err := GetAgentFeatures(antreaNamespace); err != nil {
 			tb.Fatalf("Cannot determine if %s is enabled in the Agent: %v", feature, err)
 		} else if !featureGate.Enabled(feature) {
 			tb.Skipf("Skipping test because %s is not enabled in the Agent", feature)
 		}
 	}
 	if checkController {
-		if featureGate, err := data.GetControllerFeatures(antreaNamespace); err != nil {
+		if featureGate, err := GetControllerFeatures(antreaNamespace); err != nil {
 			tb.Fatalf("Cannot determine if %s is enabled in the Controller: %v", feature, err)
 		} else if !featureGate.Enabled(feature) {
 			tb.Skipf("Skipping test because %s is not enabled in the Controller", feature)
 		}
 	}
 }
 
-func ensureAntreaRunning(tb testing.TB, data *TestData) error {
-	tb.Logf("Applying Antrea YAML")
+func EnsureAntreaRunning(data *TestData) error {
```

Comment:
nit: why it needs to be public?

Reply:
my bad, actually no need to make it public.